test/db/io/http

```diff
@@ -15,3 +15,47 @@ EXPECT=<<EOF
 EOF
 RUN
 
+# NAME=r2 remote webserver
```

Comment:
why is this commented?

Reply:
i can remove it, i was about to add more http tests. but i think its fine to go one by one :D